classes/i18n/LocaleConversion.php

```diff
@@ -24,18 +24,41 @@
 namespace PKP\i18n;
 
 use Illuminate\Support\Arr;
+use PKP\core\Core;
 use PKP\facades\Locale;
 
 class LocaleConversion
 {
+    /**
+     * Get ISO639-2b array
+     */
+    protected static function getISO6392b(): array
+    {
+        $iso6392BFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/vendor/sokil/php-isocodes-db-i18n/databases/iso_639-2.json';
+        if (!file_exists($iso6392BFile)) {
+            return [];
+        }
+        return json_decode(file_get_contents($iso6392BFile), true);
+    }
+
     /**
      * Translate the ISO 2-letter language string (ISO639-1) into a ISO compatible 3-letter string (ISO639-2b).
      */
     public static function get3LetterFrom2LetterIsoLanguage(?string $iso2Letter): ?string
     {
         assert(strlen($iso2Letter) === 2);
```

Comment:
Unrelated to this task, but I think these asserts can be removed now.
If they are really important, they should be turned into an `InvalidArgumentException`

Reply:
Removed...